line/backend/src/functions/webhook.ts

```diff
@@ -6,6 +6,86 @@ const { MessagingApiClient } = messagingApi;
 
 const client = new MessagingApiClient(config.line.messagingApiClient);
 
+type User = {
+  user_id: string;
+  name: string;
+};
+
+type TalkHistories = {
+date: string;
+user: User;
+message: string;
+};
+
+async function sendTalkRequest(talkHistories: TalkHistories[]): Promise<void> {
+
+
+
+  const requestMessage = {
+    talkHistories,
+    actionKind: 1,
+  };
+
+  try {
+    const response = await fetch("http://localhost:3000/backend/v1/BackendService/Talk", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/protobuf",
+      },
+      body: JSON.stringify(requestMessage),
+    });
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+
+    console.log("Response:", response);
+  } catch (error) {
+    console.error("Error:", error);
+  }
+}
+
+function parseTalkHistories(talk: string, hostUserName:string): TalkHistories[] {
+const rows = talk.split("\n");
+const TalkHistories: TalkHistories[] = [];
+let talkDate: string | null = null;
+
+rows.forEach(row => {
+  const trimmedRow = row.trim();
+  // 日付
+  const dateMatch = trimmedRow.match(/^(\d{4}\/\d{2}\/\d{2})/);
+  if (dateMatch) {
+  talkDate = dateMatch[1].replace(/\//g, "-"); // YYYY-MM-DD
+  return;
```

Comment:
戻り値が `TalkHistories[]` であるので、配列を返す必要があります
```suggestion
  return [];
```

何も書かない場合はundefinedになります